docs/configuration/arguments.md

```diff
@@ -188,6 +188,25 @@ prefix these flags with `distributor.ha-tracker.`
 
 Flags for configuring KV store based on memberlist library (works only for the [hash ring](../architecture.md#the-hash-ring), not for the HA Tracker).
```

Comment:
I think this should be moved below, before the listing of all flags (right before `- memberlist.nodename`).

Reply:
Left the warning on top, moved "Flags ..." closer to flags.

Reply:
Very good, thanks!